cmd/info.go

```diff
@@ -18,6 +17,10 @@ const (
 	JSON Format = "json"
 )
 
+var (
```

Comment:
💡 Thoughts/observation please:

Please note, depending on the usage, you can declare them as a var and treat them as constants in terms of not mutating them:

This makes the length of the collection as fixed at compile time and prevents resizing. If you want to ensure immutability.

```*.go
var supportedDeploymentTypes = [...]string{
	"helm",
	"kustomize",
	"manifests",
}
```

OR, might be your specific equivalent rewrite.

```*.go
var supportedDeploymentTypes = []string{
	"helm",
	"kustomize",
	"manifests",
}
```

Reply:
changed to your first suggestion!